src/datasource/hkdatasource.js

```diff
@@ -584,6 +584,53 @@ class HKDatasource
     });
   }
 
+  /**
+ * @callback GetTrailCallback
+ * @param {string} err An error object that indicate if the operation was succesful or not
+ * @param {object} trail 
+ */
+/**
+ * Fetch trail
+ *
+ * @param {string} trailId The trail id to retrieve their nested entities.
+ * @param {GetTrailCallback} callback Callback with the entities
+ */
+
+fetchTrail(trailId, callback = () => {})
```

Comment:
Please fix the indentation of this method

Reply:
Fixed